templates/faq.html

```diff
@@ -22,7 +23,7 @@
 
 <header>
 <h1>שאלות נפוצות</h1>
-<div w3-include-html="menu.html"></div>
+<div w3-include-html="/templates/menu.html"></div>
```

Comment:
I think it should be `static/menu.html` (this is how you did it elsewhere)

Reply:
cd2e2fce690a55f3b5580846cc97d2fd385fe902